### ISSUE TITLE
How to modify the search bar and the suggest list?

### ISSUE THREAD

**Issue by manhDat0301** [fysoul17/google_maps_place_picker#39]
![1](https://user-images.githubusercontent.com/57827919/82999736-a88fe400-a033-11ea-9c7b-7da8b8126af3.png)

**Comment by fysoul17** [comment]
Hi, 

Unfortunately, as it is the main feature of the package, it does not provide the feature of hiding the search bar. 

One good quick way is to fork it and just remove Autocomplete component on your side.

However, I will consider putting optional parameter to hide it later.

**Comment by giorgio79** [comment]
Just suggested a related feature https://github.com/fysoul17/google_maps_place_picker/issues/50

**Comment by WianSnyman** [comment]
Is there a way to increase the padding of the search bar so that is appears a little lower?

**Comment by fysoul17** [comment]
No, it is actually positioned 'inside' of default AppBar which you cannot modify the height unless we switch to SliverAppBar.

**Comment by diego-lipinski-de-castro** [comment]
Would be nice to be able to customize at least some aspects of the search bar.

**Comment by xJon** [comment]
Would be a great feature!
